examples/prismic/components/Builtins/Blog/Blog.tsx

```diff
@@ -0,0 +1,9 @@
+'use client'
+
+type Props = {
+  content?: React.ReactNode
+}
+
+export default function Blog({ content }: Props) {
+  return content
```

Comment:
🍹 Do we need to wrap this in a React fragment? What if `React.ReactNode` is `undefined`?